15_0_Setting_Up_a_Bitcoin_Regtest.md

```diff
@@ -0,0 +1,18 @@
+# Chapter 15: Setting Up a Bitcoin Regtest
+
+While developing Bitcoin applications, you might want to use your applications isolated from any public Blockchain such as the Mainnet or the Testnet.
+You can create a Blockchain from scratch using the Regtest, with one main advantage over Testnet: you choose when to create new blocks, so you have complete control over the environment.
+
+## Objectives for This Chapter
+
+After working through this chapter, a developer will be able to:
+
+   * Create your own Blockchain from scratch
+   * Generate/mine blocks and get the rewards
+   * Get balance and interact with Bitcoind in a private way
+
+## Table of Contents
+
+  * [Section One: Starting the Regtest](15_1_Starting_the_Regtest.md)
```

Comment:
This name is confusing and misleading. I suggest `Starting a Node on the Regtest Network`